src/AWS.Deploy.Common/Exceptions.cs

```diff
@@ -8,148 +8,230 @@
 
 namespace AWS.Deploy.Common
 {
-    public class ProjectFileNotFoundException : Exception
+    [AWSDeploymentExpectedException]
+    public class DeployToolException : Exception
```

Comment:
Since we are never directly initializing an object of `DeployToolException`, can we mark this as an abstract class ?

Reply:
Done